conventions/services/recapitulatif.py

```diff
@@ -427,6 +439,8 @@ def send_email_correction(
             email_data={
                 "convention_url": convention_url,
                 "convention": str(convention),
+                "user": str(user),
+                "bailleur": str(bailleur),
```

Comment:
@MarjolaineLB cf mon commentaire du dessus, ce que je propose est d'utiliser l'instance de convention ici 
```suggestion
                "bailleur": str(convention.bailleur.nom),
```

comme ici par exemple https://github.com/MTES-MCT/apilos/pull/985/files#diff-8320ecec0ca6f2252e9fc83d5df9f9e92359d9d03aa323ca8e29173505e565d7R258